code/controllers/configuration/entries/game_options.dm

```diff
@@ -177,6 +177,18 @@
 //MONKESTATION EDIT START
 /datum/config_entry/string/alert_epsilon
 	default = "Central Command has ordered the Epsilon security level on the station. Consider all contracts terminated."
```

Comment:
I like this, but feels pretty wordy. Alerts should be quick and snappy, not so much of a small paragraph. Ill think it over however, considering this would be nicer IC wise.